examples/04_immer/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "example",
+  "version": "0.0.0",
+  "private": true,
+  "type": "module",
+  "dependencies": {
```

Comment:
We need `"immer"` too.

Reply:
I added immer as a dependency in the examples folder, I need to also add

```
    "examples:01_counter": "DIR=01_counter vite",
    "examples:02_async": "DIR=02_async vite",
    "examples:03_actions": "DIR=03_actions vite",
    "examples:04_immer": "DIR=04_immer vite"
```

to the root package.json but it seems something desynced my local branch to this pull request and I'm hesitant to make any force pushes